docker/gaffer-gremlin/README.md

```diff
@@ -46,11 +46,20 @@ g = traversal().withRemote(
 
 A demo/example using the tinkerpop 'modern' dataset and accumulo backed Gaffer
 is available under the `example` directory. This can be ran using docker compose
-to deploy the containers then, the provided jupiter notebook demonstrates how to
-connect and some basic queries on the data using `gremlinpython`.
+to deploy the containers then, the provided jupiter notebooks demonstrate how to
+connect and some basic queries on the data using `gremlinpython` or via
```

Comment:
```suggestion
connect to your graph and some basic Gremlin queries on the data using `gremlinpython` or via
```
I think this sentence just needs a little clarity. This is just one way of doing that I think.